controllers/user/user.go

```diff
@@ -0,0 +1,117 @@
+package user
+
+import (
+	userModel "github.com/divisi-developer-poros/poros-web-backend/models/user"
+	"github.com/divisi-developer-poros/poros-web-backend/util/response"
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+
+	"net/http"
+	"strconv"
+)
+
+func GetAll(c *gin.Context) {
+	var users []userModel.User
+
+	if err := userModel.GetAll(&users); err != nil {
+		response.Res(c, "Content-Type", "application/json", "error", "failed when fetching users", http.StatusBadRequest, err)
```

Comment:
Don't put the error message on "data" field on json. Instead put all the error message on message field.